src/libraries/System.Private.CoreLib/src/System/Reflection/InvokerEmitUtil.cs

```diff
@@ -64,6 +65,11 @@ public static unsafe InvokeFunc CreateInvokeDelegate(MethodBase method)
             }
 
             // Invoke the method.
+#if !MONO
+            il.Emit(OpCodes.Call, Methods.NextCallReturnAddress()); // For CallStack reasons, don't inline target method.
+            il.Emit(OpCodes.Pop);
```

Comment:
> FWIW, in the current JIT I believe this will actually end up suppressing inlining for the remainder of the IL it sees

For my edification, what is "this" in the above statement?

Reply:
`System.StubHelpers.StubHelpers.NextCallReturnAddress()` is an intrinsic used to implement tailcalls. It has the side effect of guaranteeing that the next call-producing IL instruction will not be inlined by the JIT, so I suggested to @steveharter to use it in #69154.